### PR TITLE
Create Steelpernach.Prefab

### PR DESCRIPTION
**Pull request by mrigendraagrawal**

### CODE REVIEW

**Comment by xrtariq2594** [comment]
@mrigendraagrawal I like the idea of a variant mace. However, there are a few comments I have.

First, could you rename the prefab to be `SteelPernach.prefab`? This is to follow the capitalization style in this module.

The rest of the comments are in the file.

File: steelpernach.prefab [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "parent" : "engine:iconItem",
+    "Item" : {
+        "icon" : "Equipment:SteelDagger",
```
Comment (xrtariq2594) [inline steelpernach.prefab]: @mrigendraagrawal Could you replace this with an icon of a Steel Pernach? Remember that the image must be compliant with the Apache 2.0 license.

File: steelpernach.prefab [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        "damageType" : "axeDamage",
+        "baseCooldownTime" : 500,
+        "cooldownTime" : 500
+    },
+    "DisplayName" : {
+        "name" : "Pernach",
```
Comment (xrtariq2594) [inline steelpernach.prefab]: `"name" : "Steel Pernach"`

File: steelpernach.prefab [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        "description" : "a type of flanged mace made out of steel."
```
Comment (xrtariq2594) [inline steelpernach.prefab]: Make sure to capitalize the the "a".

**Comment by xrtariq2594** [comment]
Once these are resolved, I'll accept this and merge.

**Comment by Cervator** [comment]
Cool weapon name, hadn't heard that one before :-)

It appears the image has a white background rather than transparent, mind fixing that? Otherwise it'll appear in-game as a white square with the weapon drawn on it. Did you draw it yourself? We try to keep separate attribution for artwork to make sure we credit the artists.

Also, based on your repo it seems like you're adding files through the GitHub web UI? That tends to be a pretty awkward way to work. Using a source workspace + `gradlew fetchModuleEquipment` + `gradlew idea` to set up IntelliJ to know about the module is the way to go and also allows you to test it in-game (even submit a screenshot of yourself wielding the weapon!)

https://github.com/MovingBlocks/Terasology/wiki/Dev-Setup should help you get started there, and setting up to run from source is another valid GCI task :-)

**Comment by smsunarto** [comment]
PR closed due to inactivity - @mrigendraagrawal Please commit the required changes and re-open if you are willing to continus this PR.

**Pull request closed without merging**